source/hooks/usePrice.ts

```diff
@@ -23,11 +23,20 @@ export const usePrice = () => {
 
     currency = currency.toUpperCase();
 
+    const quantityOfZerosAfterDot = -Math.floor(Math.log10(value) + 1);
+
+    const fractionValidation =
+      quantityOfZerosAfterDot !== -0 && quantityOfZerosAfterDot <= 6;
+
     const currencySymbol = getSymbolFromCurrency(currency);
 
     const formattedValue = value.toLocaleString(navigator.language, {
-      minimumFractionDigits: precision,
-      maximumFractionDigits: precision,
+      minimumFractionDigits: fractionValidation
```

Comment:
Pls, double-check this section of the code. There is an erratic behavior when switching from network to network, that causes pali UI to break  with the following error, the Home component is the one to break due to minimumFractionDigit value being out of range during the network switch:
![image](https://user-images.githubusercontent.com/21185537/197846169-773bbd0f-62e2-41d7-883b-a5d14b2d0281.png)

Reply:
Issue already fixed in new commits